scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -134,7 +134,7 @@ export class LaneShowCmd implements Command {
   alias = '';
   options = [
     ['j', 'json', 'show the lane details in json format'],
-    ['r', 'remote', 'show the lane from remote'], // @david what does this mean?
+    ['r', 'remote', 'show the remote head of the provided lane'],
```

Comment:
Maybe the "remote heads"? (each component has its own head..)

Reply:
it's really the conceptual 'head' of the lane
I'm worried that 'heads' will be confusing, and if we put something like 'head of the provided lane's components' it's not clear it's their head on that lane (and not general head on main).
I think it's clear what the lane head is personally